CONTRIBUTING.md

```diff
@@ -48,7 +56,7 @@ All of these are automatically built and deployed by [GitHub Actions](https://gi
 
   Refer to [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) for more details.
 
-* Install [pre-commit](https://pre-commit.com/#install) to automatically ensure that your commits comply with our code style for both C++ and JavaScript. This saves time reviewing so we don't have to point out nitpicky style issues. Once you have pre-commit installed on your computer, set it up in your local Git repository:
+* <a name="pre-commit"></a> Install [pre-commit](https://pre-commit.com/#install) to automatically ensure that your commits comply with our code style for both C++ and JavaScript. This saves time reviewing so we don't have to point out nitpicky style issues. Once you have pre-commit installed on your computer, set it up in your local Git repository:
```

Comment:
Why not using a heading here rather than making an anchor this way?

Reply:
The trade-off was adding that anchor here or moving some more text around. I now moved the `pre-commit` instructions into their own section; I think I like that better, as its more clearly visible. Thanks for the suggestion! :+1: